tailscale/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN \
         ipcalc=1.0.1-r0 \
         iproute2=5.17.0-r0 \
         iptables=1.8.8-r1 \
-        nginx=1.22.0-r0 \
+        nginx=1.22.0-r1 \
```

Comment:
This is not related to this PR, please revert it.

Reply:
Done.

Reply:
@frenck Are you going to fix this nginx dep bump in another PR then?

Builds fail without it due to:
```
ERROR: unable to select packages:
  nginx-1.22.0-r1:
    breaks: world[nginx=1.22.0-r0]
```

Reply:
I have created https://github.com/hassio-addons/addon-tailscale/pull/109 for that purpose. Hopefully that helps!

Reply:
> @frenck Are you going to fix this nginx dep bump in another PR then?
> 
> 
> 
> Builds fail without it due to:
> 
> ```
> 
> ERROR: unable to select packages:
> 
>   nginx-1.22.0-r1:
> 
>     breaks: world[nginx=1.22.0-r0]
> 
> ```

Sure, when I have time.

Reply:
> Sure, when I have time.

No problem! Perhaps you missed my [other comment](https://github.com/hassio-addons/addon-tailscale/pull/108#discussion_r963022505) above. I tested https://github.com/hassio-addons/addon-tailscale/pull/109 build locally and it appears to be good to go.

Reply:
I didn't mis it, I am abroad not able to work on things.